### PR TITLE
Update Core

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

add some manual rusty reinf walls, improve centcom notes, and other misc things.

Add a custom parallax background reusing aspid's (aspid is gone now) and adding a custom planet in the background that i drew.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

image

image

### CODE REVIEW

**Comment by Kadeo64** [comment]
considering half the time you'll be staring at space because the station is gone i'd say this is a pretty good change

**Comment by Krunklehorn** [comment]
Beautiful!

**Comment by K-Dynamic** [comment]
Gas giants aren't real according to NT posters :trollface:

File: Resources/Textures/Parallaxes/attributions.yml [hunk 1]
```diff
@@ -12,3 +12,8 @@
   license: "CC-BY-NC-SA-3.0"
   copyright: "made by brainfood1183 (github) for ss14"
   source: "https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Parallaxes/gas_giant.png"
+
+  - files: ["core_planet.png"]
+  license: "CC-BY-NC-SA-3.0"
+  copyright: "Drawn by Ubaser"
+  source: "https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Parallaxes/core_planet.png"
```
Comment (metalgearsloth) [inline Resources/Textures/Parallaxes/attributions.yml:19]:
```suggestion

- files: ["core_planet.png"]
  license: "CC-BY-NC-SA-3.0"
  copyright: "Drawn by Ubaser"
  source: "https://github.com/space-wizards/space-station-14/blob/master/Resources/Textures/Parallaxes/core_planet.png"

```

**Review by EmoGarbage404 — DISMISSED** [review]
The planet in the background has some incredibly noisy pixel artifacting. It looks like a much higher resolution was scaled down and it looks quite nasty in other spots. The sun (?) Behind it is also at a really weird scale which just looks kinda strange. Beyond that, the light from the sun has some really strong banding (though that might be from the screenshot)

**Comment by metalgearsloth** [comment]
Also you'd think the sun should be a different parallax layer.

**Comment by IProduceWidgets** [comment]
core_planet

**Comment by UbaserB** [comment]
I dont know why it's sideways, but i fixed up the planet

**Comment by brainfood1183** [comment]
untitled-405 The shading has very rough edges, almost like the new york skyline. L shaped edges typically not good and you should be able to smooth it out with an additional pixel.


untitled-406
The edges of the planet are very rough in places like here.

untitled-407
This edge between the star and planet looks like it was cut out with an axe and isnt very even.

**Pull request merged**